### ISSUE TITLE
Throw a "Not Found" exception for routes that doesn't exist

### ISSUE THREAD

**Issue by lucasRolff** [laravelista/loki#7]
Would it be possible to throw a normal 404 not found exception (NotFoundHttpException I think?) for routes that don't exist.

Currently it will result in something like:

```
[2018-09-13 19:19:08] local.ERROR: Call to a member function getName() on null (View: /resources/views/items/nav.blade.php) (View: /resources/views/items/nav.blade.php) (View: /resources/views/items/nav.blade.php) {"exception":"[object] (ErrorException(code: 0): Call to a member function getName() on null (View: /resources/views/items/nav.blade.php) (View: /resources/views/items/nav.blade.php) (View: /resources/views/items/nav.blade.php) at /vendor/laravelista/loki/src/UrlGenerator.php:74, ErrorException(code: 0): Call to a member function getName() on null (View: /resources/views/items/nav.blade.php) (View: /resources/views/items/nav.blade.php) at /vendor/laravelista/loki/src/UrlGenerator.php:74, ErrorException(code: 0): Call to a member function getName() on null (View: /resources/views/items/nav.blade.php) at /vendor/laravelista/loki/src/UrlGenerator.php:74, Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Call to a member function getName() on null at /vendor/laravelista/loki/src/UrlGenerator.php:74)
```

It happens when you do something like this:

```
@foreach(config('loki.supportedLocales') as $locale)
    <li class="nav-item"><a rel="alternate" hreflang="{{ $locale }}" class="nav-link" href="{{ URL::getLocalizedRoute($locale) }}"><img src="{{ Asset::url("/assets/images/$locale.png") }}" /></a></li>
@endforeach
```

**Comment by mabasic** [comment]
Makes sense.

**Comment by mabasic** [comment]
Resolved in [`3.0.2`](https://github.com/laravelista/loki/releases/tag/3.0.2) release.

**Comment by lucasRolff** [comment]
Thank you @mabasic ! You're awesome
